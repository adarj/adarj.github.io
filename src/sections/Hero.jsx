import React from 'react';
import MainHeading from '../components/MainHeading';
import heroBg from '../images/hero-bg.jpg';

const title = "AKIL DARJEAN";
const subtitle = "SOFTWARE DEVELOPER";
const links = {
  email: "mailto:akildarjean@protonmail.com",
  github: "https://github.com/adarj",
  linkedin: "https://linkedin.com/in/adarj"
};

const Hero = () => (
  <MainHeading image={heroBg}
               title={title}
               subtitle={subtitle}
               {...links} />
);

export default Hero;
