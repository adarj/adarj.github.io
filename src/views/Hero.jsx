import React from "react"
import LazyHero from "react-lazy-hero"
import heroBg from "../images/hero-bg.jpg"

const Hero = ({ children }) => (
  <LazyHero imageSrc={heroBg} color="#000" opacity={0.6} minHeight="100vh">
    {children}
  </LazyHero>
)

export default Hero
