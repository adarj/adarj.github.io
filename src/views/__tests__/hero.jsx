import React from "react"
import renderer from "react-test-renderer"
import Hero from "../Hero"

describe("Hero", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Hero>
          <h1>Hello, world!</h1>
        </Hero>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
