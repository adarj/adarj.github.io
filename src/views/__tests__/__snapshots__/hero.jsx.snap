// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero renders correctly 1`] = `
<div
  className="sc-bwzfXH ivqze"
>
  <div
    className="sc-bdVaJa sc-htpNat doXHcr"
    style={
      Object {
        "backgroundPositionY": "center",
      }
    }
  />
  <div
    className="sc-bdVaJa sc-bxivhb kXyaSD"
    color="#000"
    opacity={0.6}
  >
    <div>
      <h1>
        Hello, world!
      </h1>
    </div>
  </div>
</div>
`;
